docs/data-sources/project_issue.md

```diff
@@ -0,0 +1,94 @@
+- **downvotes** (Number) The number of downvotes the issue has received.
+- **due_date** (String) The due date. Date time string in the format YYYY-MM-DD, for example 2016-03-11.
+**Note:** removing a due date is currently not supported, see https://github.com/xanzy/go-gitlab/issues/1384 for details.
+- **epic_id** (Number) ID of the epic to add the issue to. Valid values are greater than or equal to 0.
+- **epic_issue_id** (Number) The ID of the epic issue.
+- **external_id** (String) The external ID of the issue.
+- **human_time_estimate** (String) The human-readable time estimate of the issue.
+- **human_total_time_spent** (String) The human-readable total time spent of the issue.
+- **issue_id** (Number) The instance-wide ID of the issue.
+- **issue_link_id** (Number) The ID of the issue link.
+- **issue_type** (String) The type of issue. Valid values are: `issue`, `incident`, `test_case`.
+- **labels** (Set of String) The labels of an issue.
+- **links** (Map of String) The links of the issue.
+- **merge_request_to_resolve_discussions_of** (Number) The IID of a merge request in which to resolve all issues. This fills out the issue with a default description and mark all discussions as resolved. When passing a description or title, these values take precedence over the default values.
+- **merge_requests_count** (Number) The number of merge requests associated with the issue.
+- **milestone_id** (Number) The global ID of a milestone to assign issue. To find the milestone_id associated with a milestone, view an issue with the milestone assigned and use the API to retrieve the issue's details.
+- **moved_to_id** (Number) The ID of the issue that was moved to.
+- **references** (Map of String) The references of the issue.
+- **state** (String) The state of the issue. Valid values are: `opened`, `closed`.
+- **subscribed** (Boolean) Whether the authenticated user is subscribed to the issue or not.
+- **task_completion_status** (List of Object) The task completion status. It's always a one element list. (see [below for nested schema](#nestedatt--task_completion_status))
+- **time_estimate** (Number) The time estimate of the issue.
+- **title** (String) The title of the issue.
+- **total_time_spent** (Number) The total time spent of the issue.
+- **updated_at** (String) When the issue was updated. Date time string, ISO 8601 formatted, for example 2016-03-11T03:45:40Z.
+- **upvotes** (Number) The number of upvotes the issue has received.
+- **user_notes_count** (Number) The number of user notes on the issue.
+- **web_url** (String) The web URL of the issue.
+- **weight** (Number) The weight of the issue. Valid values are greater than or equal to 0.
+
+<a id="nestedatt--task_completion_status"></a>
+### Nested Schema for `task_completion_status`
+
+Read-Only:
+
+- **completed_count** (Number)
+- **count** (Number)
```

Comment:
Wouldn't `count` collide with the `count` meta-argument?

https://www.terraform.io/language/meta-arguments/count

Reply:
I'm not 100% sure, but I think it's okay. Other prominent provider use it, too in nested schemas at least:

* https://cs.github.com/hashicorp/terraform-provider-aws/blob/8d7b2519ddcfd2a4d6e9b907cee38aafe302733a/internal/service/ec2/instance_type_data_source.go?q=language%3Ago+org%3Ahashicorp+%5C%22count%5C%22+repo%3Ahashicorp%2Fterraform-provider-aws#L126
* https://cs.github.com/hashicorp/terraform-provider-aws/blob/8d7b2519ddcfd2a4d6e9b907cee38aafe302733a/internal/service/wafv2/web_acl.go?q=language%3Ago+org%3Ahashicorp+%5C%22count%5C%22+repo%3Ahashicorp%2Fterraform-provider-aws#L111

Also I think you can't access `count` as an attribute on a resource in case it used `count = N` (even though I think it used, too) - You'll get something like:

```
The special "count" attribute is no longer supported after Terraform v0.12. Instead, use length(gitlab_project.for_user) to count resource instances.
```

I think it's okay to use here, because it's a nested resource and not an resource block level and because it's read-only.

Please correct me if I'm wrong :D

Reply:
Aaaaaaaaaaaah oops I thought I deleted this comment. Yeah I came to that conclusion when I realized it was a nested schema. Thanks for double checking.